test/functional/uuid/jsonb.ts

```diff
@@ -7,10 +7,16 @@ import {createTestingConnections, closeTestingConnections} from "../../utils/tes
 describe("uuid type", () => {
 
     let connections: Connection[];
-    before(async () => connections = await createTestingConnections({
-        entities: [Record],
-        enabledDrivers: ["postgres"] // because only postgres supports jsonb type
-    }));
+    before(async () => {
+        connections = await createTestingConnections({
+            entities: [Record],
+            enabledDrivers: ["postgres"] // because only postgres supports uuid type
+        });
+
+        connections.forEach(async connection => {
```

Comment:
also you probably know that you are not awaiting here execution of `query` ? Because if you want to await it here you need to do: `await Promise.all(connections.map(async connection => ... ));` instead